src/screens/manage-token/manage-token.tsx

```diff
@@ -176,31 +176,39 @@ export class ManageTokenComponent extends React.Component<
         let blockchainToken;
 
         if (isContractAddress) {
-            const getTokenInfo = await Promise.all([
-                fetch(
-                    CONFIG.tokensUrl +
-                        `${blockchain.toLocaleLowerCase()}/${this.state.fieldInput.toLocaleLowerCase()}.json`
-                ),
-                getBlockchain(blockchain)
-                    .getClient(this.props.chainId)
-                    .tokens[tokenType].getTokenInfo(this.state.fieldInput)
-            ]);
-            if (getTokenInfo[0].status === 200) {
-                staticToken = await getTokenInfo[0].json();
+            try {
+                const getTokenInfo = await Promise.all([
+                    fetch(
+                        CONFIG.tokensUrl +
+                            `${blockchain.toLocaleLowerCase()}/${this.state.fieldInput.toLocaleLowerCase()}.json`
+                    ),
+                    getBlockchain(blockchain)
+                        .getClient(this.props.chainId)
+                        .tokens[tokenType].getTokenInfo(this.state.fieldInput)
+                ]);
+                if (getTokenInfo[0].status === 200) {
+                    staticToken = await getTokenInfo[0].json();
+                }
+                blockchainToken = getTokenInfo[1];
+            } catch {
+                //
```

Comment:
We should handle errors, maybe display an error message for the user.

Reply:
Lower in the code we handle this. Since there are multiple versions of getting a token if non works we display an error message to user.